src/WSRequestHandler_SceneItems.cpp

```diff
@@ -402,16 +403,18 @@ RpcResponse WSRequestHandler::ResetSceneItem(const RpcRequest& request) {
 * @since 0.3
 */
 RpcResponse WSRequestHandler::SetSceneItemRender(const RpcRequest& request) {
-	if (!request.hasField("source") ||
-		!request.hasField("render"))
+	if (!request.hasField("render") ||
+		(!request.hasField("source") && !request.hasField("item"))
```

Comment:
It would be cleaner to have everything on one line. To make it more readable, you can write it like this:

```cpp
bool hasSourceOrItemParameter = ( request.hasField("source") || request.hasField("item") );
if (!request.hasField("render") || !hasSourceOrItemParameter) {
```